src/toggle/toggle.component.ts

```diff
@@ -109,6 +109,16 @@ export class Toggle extends Checkbox {
 		return this._offValues.value;
 	}
 
+	/**
+	 * Sets the aria labelledby on the `toggle`s label element.
+	 */
+	@Input() ariaLabelledby: string;
```

Comment:
I believe the checkbox component has these inputs, does setting the inputs still work without having these?

Reply:
Thanks, we have overlooked it.
We are removing them from toggle as we have them in checkbox component